__tests__/app.test.js

```diff
@@ -12,6 +12,26 @@ afterAll(() => {
     connection.end();
 })
 
+describe('/api', () => {
+    test('GET: 200 Responds with an object describing all the available endpoints', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({ body }) => {
+            const availableEndpoints = Object.keys(body.endpoints)
+            availableEndpoints.forEach((endpoint) => {
+                expect(endpoint.startsWith('GET') || endpoint.startsWith('POST') || endpoint.startsWith('DELETE') || endpoint.startsWith('PATCH')).toBe(true);
+                expect(body.endpoints[endpoint]).toMatchObject({
+                    description: expect.any(String),
+                    queries: expect.any(Array),
+                    requestFormat: expect.any(Object),
+                    exampleResponse: expect.any(Object)
+                })
```

Comment:
fine test, if a little complex! Might make it a little brittle later, but sure you'll be able to figure it out if it does fail in the future.  ps. you can test for equality to the endpoints.json required in.